values.yaml

```diff
@@ -120,4 +106,29 @@ relay:
 
   ## Secrets don't like '-'
   secrets:
-    relayCogToken: 
+    relayCogToken:
+
+nginx:
+  name: nginx
+  replicaCount: 1
+  image:
+    repository: nginx
+    tag: 1.11
+  resources:
+    # limits:
+    #   cpu: 100m
+    #   memory: 128Mi
+    requests:
+      cpu: 100m
+      memory: 128Mi
+  serviceType: LoadBalancer
+  ingress:
```

Comment:
I think packaged nginx as a whole is a great idea here to simplify interacting with Cog.

My suggestion was that the `ingress.enable` value be top level rather than under `cog` in the `values.yml`. The ingress itself would point to nginx but be determined by the larger decision of whether you want external traffic to route to Cog.